hack/keys/create.sh

```diff
@@ -30,18 +30,21 @@ if [ -n ${OS_ROOT} ]; then
   SIGNER_SERIAL=${OS_ROOT}/master/ca.serial.txt
 fi
 
+function createCertificate {
+  name=$1
+  hostnames=$2
+  mkdir -p $name
+  oadm ca create-server-cert --cert=$name/$name.cert --key=$name/$name.key --hostnames=$hostnames --signer-cert=${SIGNER_CERT} --signer-key=${SIGNER_KEY} --signer-serial=${SIGNER_SERIAL}
+  cat $name/$name.cert $name/$name.key > $name/$name.pem
+
+  echo "`openssl x509 -in $name/$name.cert; cat $name/$name.key`" > $name/$name-noca.pem
```

Comment:
( openssl x509 -in $name/$name.cert; cat $name/$name.key ) > $name/$name-noca.pem